### ISSUE TITLE
PHPDocumentor generates 29 Docblock errors, which I'm happy to fix if you want them fixed :-)

### ISSUE THREAD

**Issue by yourpropertyexpert** [amnuts/opcache-gui#54]
If you run this code through PHPDocumentor, it generates 29 errors (which is a tiny number.)

I'd be happy to "fix them" by adding comments to the code...

... but I only want to spend time on this if it's actually of interest and something you'd be happy to merge if I did...

**Comment by amnuts** [comment]
Thanks for the input @yourpropertyexpert.  If you ask me, 29 errors is 29 too many. 😉 

I'm more than happy for you to submit a pull request if you'd like.  It's probably best against the `v3` branch, though, as I'm working on that code at the moment and the PHP got restructured and the namespace changed.

**Comment by amnuts** [comment]
Actually, you can use the `master` branch now as I merged in the changes from `v3` yesterday and released the new version.

**Comment by yourpropertyexpert** [comment]
Thanks. I'll try to do that this week...

**Comment by yourpropertyexpert** [comment]
The good news....

Your release last weekend dropped the number from 29 down to 3... so I had very little to do.

The bad news....

I've only been able to get rid of one of the outstanding 3.

The other two are to do with limitations in PHPDocumentor 2 (the current live version), which can't cope with nullable type hinting. This is on the list for PHPDocumentor 3... but that's only in RC status, and I'm reporting issues there...

... but your Docblocks in the files responsible for those two errors look like they should pass :-)


I've submitted a pull request into your codebase for the one I could fix.
